docs/CLI.md

```diff
@@ -43,3 +43,9 @@ Options
   --yes, -y      confirm non-interactively
   --unmount, -u  unmount on success
 ```
+
+Debug mode
+----------
+
+You can set the `DEBUG` environment variable to make the Etcher CLI print error
```

Comment:
I wonder if it should be more specific, like `ETCHER_CLI_DEBUG`? The user might have `DEBUG` set in their environment variables for other reasons, without wanting to debug the Etcher CLI? ;-)